pkg/jobs/ansible/job.go

```diff
@@ -62,6 +68,12 @@ func Job(client client.Client, curator *clustercuratorv1.ClusterCurator) error {
 			case "upgrade":
 				hooks = curator.Spec.Upgrade
 		*/
+	case "installPosthook":
+		posthook = curator.Spec.Install.Posthook
```

Comment:
```
if (desiredCuration == "installPosthook" && curator.Spec.Install.Posthook != nil) || (desiredCuration == "upgradePosthook" && curator.Spec.Upgrade.Posthook != nil) {
```

Do we need the above checks around here as well?

Reply:
No, I think it should be fine here. I had that check here https://github.com/stolostron/cluster-curator-controller/pull/159/files#diff-bf83b388c4c87db8c671f7e5728bec15a9e587d04bf147cb7d061c5b585bb0c5R247 because the install/upgrade posthook cases are combined to reuse the code. For the anisble job code they are separated.